src/applications/vaos/tests/appointment-list/components/CommunityCareAppointmentDetailsPage.unit.spec.jsx

```diff
@@ -428,53 +429,39 @@ describe('VAOS <CommunityCareAppointmentDetailsPage>', () => {
         .getAttribute('href')
         .replace('data:text/calendar;charset=utf-8,', ''),
     );
-    const tokens = ics.split('\r\n');
+    const tokens = getICSTokens(ics);
 
-    expect(tokens[0]).to.equal('BEGIN:VCALENDAR');
-    expect(tokens[1]).to.equal('VERSION:2.0');
-    expect(tokens[2]).to.equal('PRODID:VA');
-    expect(tokens[3]).to.equal('BEGIN:VEVENT');
-    expect(tokens[4]).to.contain('UID:');
+    expect(tokens.get('BEGIN').includes('VCALENDAR')).to.be.true;
+    expect(tokens.get('VERSION')).to.equal('2.0');
+    expect(tokens.get('PRODID')).to.equal('VA');
+    expect(tokens.get('BEGIN').includes('VEVENT')).to.be.true;
```

Comment:
FYI, Chai has a lot of different [helpers](https://www.chaijs.com/api/bdd/#method_include), so you can do things like `expect(tokens.get('BEGIN')).to.include('VEVENT')`, which is a little nicer to read.

No need to update these unless you want to, just wanted to point it out, since we don't always make full use of the different Chai assertions.